connectivity/connectivity-demos/src/com/axonivy/connectivity/soap/client/CustomLogFeature.java

```diff
@@ -0,0 +1,64 @@
+package com.axonivy.connectivity.soap.client;
+
+import org.apache.cxf.endpoint.Client;
+import org.apache.cxf.frontend.ClientProxy;
+import org.apache.cxf.interceptor.Fault;
+import org.apache.cxf.message.Message;
+import org.apache.cxf.phase.AbstractPhaseInterceptor;
+import org.apache.cxf.phase.Phase;
+import org.apache.http.HttpStatus;
+import ch.ivyteam.ivy.environment.Ivy;
+import ch.ivyteam.ivy.webservice.exec.feature.WebServiceClientFeature;
+import ch.ivyteam.ivy.webservice.exec.feature.WebServiceClientFeatureContext;
+
+/**
+ * Demonstrates the setup of custom interceptors that track
+ * the messages going out to the remote service and their responses.
+ */
+public class CustomLogFeature  implements WebServiceClientFeature {
+
+  private static final String EXCHANGE_ID_KEY = "exchangeId";
+
+  @Override
+  public void initialize(WebServiceClientFeatureContext context) {
+    Client proxy = ClientProxy.getClient(context.getBindingProvider());
+    proxy.getOutInterceptors().add(new CxfSendInterceptor(Phase.SEND_ENDING));
+    proxy.getInInterceptors().add(new CxfReceiveInterceptor(Phase.RECEIVE));
+    Ivy.log().info("call initiated with custom interceptors provided by "+this);
+  }
```

Comment:
main part of the demo: showing our customers how to add interceptors just for this webservice-client ... rather than using the global Bus instance.